app/controllers/api/v1/site_controller.rb

```diff
@@ -51,6 +51,46 @@ def studies
         @studies = Study.viewable(current_api_user)
       end
 
+      swagger_schema :DirectoryListingDownload do
+        property :name do
+          key :type, :string
+          key :description, 'Name of remote GCS directory containing files'
```

Comment:
This acronym is worth expanding on first use in an endpoint:

```suggestion
          key :description, 'Name of remote Google Cloud Storage (GCS) directory containing files'
```